open_discussions_api/client.py

```diff
@@ -1,12 +1,8 @@
 """open-discussions api client"""
-import time
-
-import jwt
 import requests
 
 from .users.client import UsersApi
-
-EXPIRATION_DELTA_SECONDS = 60 * 60
+from .utils import get_token
```

Comment:
this my first review in this project and I do not know if we used different conventions, but I thing we should avoid to use relative imports
But please disregard if the convention is that it is OK to use them.

Reply:
We use them in `edx-api-client`, which I based this project off of, so the style carried over unintentionally. I'll fix it.

Reply:
Actually, even worse, this project was inconsistent. Fixed now.